tests/sentry/api/endpoints/test_relay_globalconfig_v3.py

```diff
@@ -39,14 +39,6 @@ def test_global_config():
     config["options"]["profiling.profile_metrics.unsampled_profiles.platforms"] = ["fake-platform"]
     config["options"]["profiling.profile_metrics.unsampled_profiles.sample_rate"] = 1.0
 
-    config["options"]["relay.metric-bucket-encodings"] = {
-        "sessions": "array",
-        "transactions": "array",
-        "spans": "array",
-        "custom": "array",
-        "profiles": "array",
-        "unsupported": "array",
-    }
```

Comment:
Not overwriting the default values results in Relay skipping serialization, meaning the normalization of options is not tested. Could we set non-default values and ensure they're tested?

Reply:
Not sure what you mean, even invalid values are accepted by Relay and serialized again. I don't think this test is actually useful, it doesn't even test if the value is correctly loaded from the options.

Reply:
The test validates whether relay (de)serializes options correctly. Since relay doesn't serialize options with default values, it's hard to validate whether relay parses the option appropriately. Is this something we'd want to cover?